onnx/defs/nn/defs.cc

```diff
@@ -741,6 +741,12 @@ void convTransposeShapeInference(InferenceContext& ctx) {
     return;
   }
 
+  int64_t group = 1;
```

Comment:
You can replace these lines with "int64_t group = getAttribute(ctx, "group", 1)".